WORKSPACE

```diff
@@ -112,3 +118,18 @@ docker_pull(
     registry = "gcr.io",
     repository = "gae-runtimes/php72_app_builder"
 )
+
+# Go Dependencies for Container Tool Command Library.
+# Import Go dependencies.
+go_repository(
```

Comment:
I think if you use gazelle with mode external="vendored" you don't need these.

Reply:
yeah i saw @aaron-prindle added after i started working on this. I will try gazelle.

Reply:
done